tests/baselines/reference/organizeImports/Unused_false_positive_module_augmentation.ts

```diff
@@ -0,0 +1,22 @@
+// ==ORIGINAL==
+
+import foo from 'foo';
+import { Caseless } from 'caseless';
+
+declare module 'foo' {}
+declare module 'caseless' {
+    interface Caseless {
+        test(name: KeyType): boolean;
+    }
+}
+// ==ORGANIZED==
+
+import 'caseless';
```

Comment:
Changing this to a side-effect import affects the emitted JS. Previously the import was elided due to being unused. With this slightly different syntax it will always be emitted.

Reply:
Ah, interesting. We can do this in declaration files (which should be the norm), but either do `import {} from 'caseless'` or just don’t touch it in TypeScript files?

Reply:
Addressed by not messing with imports for module augmentation outside of declaration files